arduino/netsender/NetSender.cpp

```diff
@@ -225,6 +225,7 @@ static unsigned long Time = 0;
 static unsigned long AlarmedTime = 0;
 static int NetworkFailures = 0;
 static int SimulatedBat = 0;
+static String Mode = "Normal";
```

Comment:
You could have constexprs for the mode strings like
```
namespace Mode {
    constexpr const char* NORMAL            = "Normal";
    constexpr const char* LOW_VOLTAGE_ALARM = "LowVoltageAlarm";
}

static String CurrentMode = Mode::NORMAL;
```

probably doesn't add that much value, but there's slightly more safety.

Reply:
Nice suggestion.